samples/search_grouding.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DynamicRetrievalMode,
+  GoogleGenerativeAI,
+} from "@google/generative-ai";
+
+async function searchGrounding() {
+  // [START search grounding]
+  const genAI = new GoogleGenerativeAI(process.env.API_KEY);
+  const model = genAI.getGenerativeModel(
+    {
+      model: "gemini-1.5-pro",
+      generationConfig: {},
+      tools: [
+        {
+          googleSearchRetrieval: {
+            dynamicRetrievalConfig: {
+              mode: DynamicRetrievalMode.MODE_DYNAMIC,
+              dynamicThreshold: 0.7,
+            },
+          },
+        },
+      ],
+    },
+    { apiVersion: "v1beta" },
+  );
+
+  const prompt =
+    "What is the Google stock today?";
+
+  const result = await model.generateContent(prompt);
+  console.log(JSON.stringify(result.response));
```

Comment:
You shouldn't need to use `JSON.stringify` here. Just `console.log(result.response)`;